.babelrc

```diff
@@ -2,6 +2,8 @@
   "presets": ["airbnb"],
   "plugins": [
     ["transform-replace-object-assign", { "moduleSpecifier": "object.assign" }],
+    "syntax-dynamic-import",
+    "dynamic-import-node"
```

Comment:
let's only add these in the "test" env; i don't think this is ever a safe transform except at the app level.